server/chat-commands/core.ts

```diff
@@ -414,11 +414,11 @@ export const commands: ChatCommands = {
 			return this.sendReply(this.tr("What?! How are you not more excited to battle?! Try /battle! to show me you're ready."));
 		}
 		if (!target) target = "randombattle";
-		return this.parse(`/search ${target}`);
+		return void this.parse(`/search ${target}`);
```

Comment:
Err, `return` already correctly propagates `this.parse`; the `void` is unnecessary.

Reply:
idk how i messed that up so bad, oops

Reply:
The `void` is actually bad because it stops the promise from being propagated like it should be.